scst-store/amr/auth-k8s-sa-autoconfiguration.md

```diff
@@ -1,24 +1,23 @@
 
-All of the package-level configuration is prefixed with a component top level key (TLK) and can also be inflenced by certain TAP level configuration, which will be mentioned in the seperate sections.
+All package-level configurations are prefixed with a component top level key (TLK) and can also be influenced by certain TAP level configurations, which will be mentioned in the separate sections.
 
 
 ## Observer
 
 Observer configuration in the TAP context is prefixed with the TLK `amr.observer`. 
-For authentication and authorization the TAP profiles will influence autconfiguration. Observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
+For authentication and authorization, the TAP profiles will influence autoconfiguration. The observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
 
-On the package level, if `auth.kubernetes_service_accounts.enable` and `auth.kubernetes_service_accounts.autoconfigure` are true, the observer package will create the following resources to set up authentication automatically in namespace `amr-observer-system`:
+On the package level, if `auth.kubernetes_service_accounts.enable` and `auth.kubernetes_service_accounts.autoconfigure` are true, the observer package will create the following resources to set up authentication automatically in the namespace `amr-observer-system`:
 
 * a `ServiceAccount` named `amr-observer-editor` that observer will use to send requests to the cloudevent handler
 * a `Secret` named `amr-observer-edit-token` of type `kubernetes.io/service-account-token` which will generate a long-lived token for the service account
-* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissons for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
+* a `ClusterRole` named `tanzu:amr:observer:edit` defining the necessary `update` permissions for all resources in `cloudevents.amr.apps.tanzu.vmware.com`
 * a `ClusterRoleBinding` named `tanzu:amr:observer:editor` binding the defined role to the service account 
 
 
-
 ## Cloudevent handler
 
 Cloudevent handler configuration in the TAP context can be found under the TLK `amr.cloudevent_handler`. This prefix is not stripped in this case. 
```

Comment:
I believe Nicholas just recently flattened the structure such that this is no longer true. 

Cloudevent handler is now under `metadata_store.cloudevent_handler` as opposed to `metadata_store.amr.cloudevent_handler`. Similarly, GraphQL is now under `metadata_store.graphql` as opposed to `metadata_store.amr.graphql`

Reply:
i changed this part to reflect these changes. i believe the values file is still not good so i posted to the slack thread again.

Reply:
tap-values has been updated to provide amr properties via the `amr` TLK. so, `amr.cloudevent_handler` and `amr.graphql` for the metadata-store package. `amr.observer` also is present for the amr-observer package. when the properties for the metadata-store are constructed to be injected into the package, properties from the `metadata_store` and `amr` TLKs are merged together. At first, with `amr` properties being injected as is, ie (as `amr.cloudevent_handler` and not `cloudevent_handler`)... essentially it functioned that declaring the amr properties within the metadata_store TLK still functioned. But now that amr properties are injected just as `cloudevent_handler`, the values schema should complain. I can make a slack post to make this clearer.